app/views/comments/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%= @comment.errors.full_messages.join(", ") %>
+
+<%= form_for @comment, url: post_comments_path(@post), html: {class: "form"}, remote: true do |f| %>
```

Comment:
Need to change the @comment to comment since it is being assigned to be "comment: Comment.new" in create_success.js.erb